corehq/apps/app_manager/tests/data/suite/fixture-to-case-selection.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<suite descriptor="Suite File">
+  <xform>
+    <resource descriptor="Form: (Module my_module) - my_form" id="form_1">
+      <location authority="local">./modules-0/forms-0.xml</location>
+      <location authority="remote">./modules-0/forms-0.xml</location>
+    </resource>
+  </xform>
+  <locale language="default">
+    <resource descriptor="Translations: Default Language" id="app_default_strings">
+      <location authority="local">./default/app_strings.txt</location>
+      <location authority="remote">./default/app_strings.txt</location>
+    </resource>
+  </locale>
+  <locale language="en">
+    <resource descriptor="Translations: English" id="app_en_strings">
+      <location authority="local">./en/app_strings.txt</location>
+      <location authority="remote">./en/app_strings.txt</location>
+    </resource>
+  </locale>
+  <detail id="m0_case_short">
+    <title>
+      <text>
+        <locale id="m0.case_short.title"/>
+      </text>
+    </title>
+    <field>
+      <header>
+        <text>
+          <locale id="m0.case_short.case_name_1.header"/>
+        </text>
+      </header>
+      <template>
+        <text>
+          <xpath function="case_name"/>
+        </text>
+      </template>
+    </field>
+  </detail>
+  <detail id="m0_case_long">
+    <title>
+      <text>
+        <locale id="m0.case_long.title"/>
+      </text>
+    </title>
+    <field>
+      <header>
+        <text>
+          <locale id="m0.case_long.case_name_1.header"/>
+        </text>
+      </header>
+      <template>
+        <text>
+          <xpath function="case_name"/>
+        </text>
+      </template>
+    </field>
+  </detail>
+  <detail id="m0_fixture_select">
+    <title>
+      <text/>
+    </title>
+    <field>
+      <header>
+        <text/>
+      </header>
+      <template>
+        <text>
+          <xpath function="my_display_column"/>
+        </text>
+      </template>
+    </field>
+  </detail>
+  <entry>
+    <command id="m0-f0">
+      <text>
+        <locale id="forms.m0f0"/>
+      </text>
+    </command>
+    <instance id="casedb" src="jr://instance/casedb"/>
+    <instance id="commcaresession" src="jr://instance/session"/>
+    <instance id="item-list:days" src="jr://fixture/item-list:days"/>
+    <session>
+      <datum detail-select="m0_fixture_select" id="fixture_value" nodeset="instance('item-list:days')/days_list/days" value="my_variable_column"/>
+      <datum detail-confirm="m0_case_long" detail-select="m0_case_short" id="case_id" nodeset="instance('casedb')/casedb/case[@case_type='cases'][@status='open'][instance('commcaresession')/session/data/fixture_value &gt; 3]" value="./@case_id"/>
```

Comment:
Sticking to the ordering defined here make it easier to read these: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/suite_xml.py#L243